tcks/resteasy-reactive/pom.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-tck-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-resteasy-reactive</artifactId>
+    <name>Quarkus - TCK - RestEasy Reactive</name>
+
+    <properties>
+        <exec.skip>${skipTests}</exec.skip>
+        <rrt.clone.skip>${exec.skip}</rrt.clone.skip>
+        <rrt.checkout.skip>${exec.skip}</rrt.checkout.skip>
+        <rrt.test.skip>${exec.skip}</rrt.test.skip>
```

Comment:
I know it's a mouthful and sorry to be that guy but I would prefer an explicit name rather than an acronym that is far from being obvious. It's not as if we would reference these properties everywhere and it would be a big win to avoid typing more letters.

Reply:
Well, I had that feeling this would come up. 😉 
The idea was indeed to reduce typing effort. You would only use those properties locally if you'd want to use a different branch or you would want to skip cloning to save a few seconds.
We already decided that we need a Readme (which I will add) and that Readme has to document the properties anyway, so IMO they could stay short.

Anyway, what do you propose? `resteasy-reactive-testsuite.*`?

Reply:
You wanted longer property names - here you go! 😉